internal/algorithm/checkpatternbasic/example_test.go

```diff
@@ -0,0 +1,81 @@
+package checkpatternbasic
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestXxx(t *testing.T) {
```

Comment:
테스트 코드 이름에 테스트 대상이 명확히 드러나도록 수정해주세요
```suggestion
func TestModel(t *testing.T)
```